bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -1534,6 +1534,17 @@ impl OlmMachine {
         }
         .into()
     }
+
+    /// Clear any in-memory caches because they may be out of sync with the
+    /// underlying data store.
+    /// 
+    /// The crypto store layer is caching olm sessions for a given device.
+    /// When used in a multi-process context this cache will get outdated.
+    /// If the machine is used by another process, the cache must be invalidating when the
+    /// main process is resumed.
+    pub async fn clear_crypto_cache(&self) {
+        self.runtime.block_on(self.inner.clear_crypto_cache())
```

Comment:
The method is async, no need for using `block_on`.

```suggestion
        self.inner.clear_crypto_cache().await
```